### DISCUSSION TITLE
Message Colors

### DISCUSSION THREAD

**Discussion by DevBlokJochem** [Minestom/Minestom#1145] [Help] [answered]
Hello, is there an easy way to include color codes/hex codes in your messages?
for example: 
§6/&6 -> gold
#ff0000 -> red

**Comment by MrlnHi** [comment, marked as answer]
For the &6 you could just parse chat messages using the LegacyComponentSerializer.AMPERSAND. (I think that's what it's called anyways)
For the #ff0000 my approach would be using a regex like "#([0-9a-f]{6})" and then parsing the first capture group of that into an int using base 16

**Reply by DevBlokJochem** [reply]
Oke. Do you have an example or something? I dont know how todo it with regex.
